loama/src/components/LoginForm.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <header>
+    <h1>{{ title }}</h1>
+    <p class="tagline" v-if="subtitle">{{ subtitle }}</p>
+  </header>
+  <form @submit.prevent="login">
+    <fieldset>
+      <legend>
+        Solid Pod URL
+        <PhQuestion :size="24" @mouseover="showPopup = true;" @mouseleave="showPopup = false" />
+      </legend>
+      <label for="solid-pod-url">
+        <PhLink :size="24" class="icon" />
+        <input type="url" id="solid-pod-url" v-model="solidPodUrl" :placeholder="defaultSolidPodUrl" />
+      </label>
+      <p v-if="showWarning" class="warning">Invalid Solid Pod URL. Please check and try again.</p>
+    </fieldset>
+    <fieldset>
+      <LoButton @click.prevent="noPod" :disabled="isLoading" class="secondary" :left-icon="PhQuestion">
+        No Pod?
+      </LoButton>
+      <LoButton type="submit" :disabled="isLoading" :right-icon="PhArrowRight">
+        <span v-if="isLoading">Loading...</span>
+        <span v-else>Login</span>
+      </LoButton>
+    </fieldset>
+  </form>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import { store } from '@/store';
+import { PhArrowRight, PhLink, PhQuestion } from '@phosphor-icons/vue';
+import LoButton from './LoButton.vue';
+
+defineProps<{ title: string, subtitle?: string }>();
+
+const showPopup = defineModel<boolean>('showPopup');
+
+const emit = defineEmits<{
+  toggleProvider: []
+}>()
+
+const solidPodUrl = ref<string>('');
+const defaultSolidPodUrl = import.meta.env.VITE_DEFAULT_IDP;
```

Comment:
This is an env variable?
Might be good to document that in the `readme.md`, can be easily missed during deployment